<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>531</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::South</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Search</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Choose dictionary:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>221</width>
        <height>32</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Slangs</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Emojis</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>English words</string>
       </property>
      </item>
     </widget>
     <widget class="QTextEdit" name="searchBar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QListWidget" name="suggestingWords">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>251</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>90</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>üîç</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>0</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>üåü</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>0</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>üî®</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="randomWord">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>330</y>
        <width>461</width>
        <height>141</height>
       </rect>
      </property>
      <property name="title">
       <string>Some random word</string>
      </property>
      <widget class="QLabel" name="word">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>381</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>15</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="def">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>431</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="definition">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>110</y>
        <width>461</width>
        <height>211</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="resetDictionary">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>470</y>
        <width>131</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset dictionary</string>
      </property>
     </widget>
     <widget class="QLabel" name="currentWord">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>50</y>
        <width>481</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>0</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Search by definition</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>10</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Words containing this in definitions</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="searchBar_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>481</width>
        <height>31</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QListWidget" name="suggestingWords2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>281</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="definition2">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>140</y>
        <width>441</width>
        <height>271</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="currentWord_2">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>70</y>
        <width>451</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Game</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>371</width>
        <height>461</height>
       </rect>
      </property>
      <property name="title">
       <string>Guess definition</string>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>100</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>160</y>
         <width>331</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>200</y>
         <width>321</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>240</y>
         <width>321</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_4">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>280</y>
         <width>321</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="guess">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>331</width>
         <height>91</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="answer">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>370</y>
         <width>331</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_7">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>330</y>
         <width>100</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Confirm</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>20</y>
        <width>371</width>
        <height>461</height>
       </rect>
      </property>
      <property name="title">
       <string>Guess word</string>
      </property>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>100</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_5">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>120</y>
         <width>21</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_6">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>170</y>
         <width>16</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_7">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>230</y>
         <width>16</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_8">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>290</y>
         <width>21</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="guess2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>331</width>
         <height>91</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="answer2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>390</y>
         <width>331</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>350</y>
         <width>100</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Confirm</string>
       </property>
      </widget>
      <widget class="QLabel" name="answer1">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>130</y>
         <width>281</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="answer1_2">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>180</y>
         <width>281</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="answer1_3">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>230</y>
         <width>281</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="answer1_4">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>290</y>
         <width>281</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Favorites</string>
     </attribute>
     <widget class="QListWidget" name="favoriteList">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>256</width>
        <height>451</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="removeFavorite">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>440</y>
        <width>151</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove from favorite</string>
      </property>
     </widget>
     <widget class="QLabel" name="definition_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>20</y>
        <width>471</width>
        <height>281</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>History</string>
     </attribute>
     <widget class="QPushButton" name="removeHistory">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>420</y>
        <width>151</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove from history</string>
      </property>
     </widget>
     <widget class="QListWidget" name="historyList">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>256</width>
        <height>451</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="clearAll">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>420</y>
        <width>151</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear all</string>
      </property>
     </widget>
     <widget class="QLabel" name="definition_3">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>10</y>
        <width>451</width>
        <height>321</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
